src/ontology/cl-edit.owl

```diff
@@ -29524,6 +29525,16 @@ AnnotationAssertion(rdfs:label obo:CL_4028006 "alveolar type 2 fibroblast cell")
 SubClassOf(obo:CL_4028006 obo:CL_0002241)
 SubClassOf(obo:CL_4028006 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002299))
 
+# Class: obo:CL_4030000 (choroidal melanocyte)
+
+AnnotationAssertion(<http://geneontology.org/formats/oboInOwl#created_by> obo:CL_4030000 "http://orcid.org/0000-0002-1773-2692")
```

Comment:
would you mind switching this to dcterms:contributor please -> think there is an effort to switch (which reminds me I need to change mine too lol)

Reply:
Is there a standard for annotating who edits/creates the term vs who requests the term?
Does requestor = contributor and editor (myself) = dcterms:contributor OR should everyone be annotated with dcterms:contributor?

Reply:
I think we are trying to move everyone to dcterms:contributor (I know that's Nico's preference at least) last we talked about this at least. Sorry for this being super confusing and all, we have recently tried to unify all these stuff but getting consensus is SUPER difficult

Reply:
Thank you, @shawntanzk! For this PR and going forward I will set both the requestor and editor to dcterms:contributor. 
Since I have not found a workflow for it, I'll plan to draft the steps for creating a new term in CL and then share with the group in an effort to get consensus.